pushjack/apns.py

```diff
@@ -129,7 +129,6 @@ def close(self):
 
     def send(self,
              ids,
-             message,
```

Comment:
You're right. For silent notifications, `message=None` should just do. I changed it back and add 2 new properties: `mutable_content` and `thread_id`. I tried the `send` function locally and all works, not sure why some tests are still failing?